.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: ci
+
+on:
+  - push
+  - workflow_dispatch
+
+env:
+  MAVEN_FLAGS: "-B --no-transfer-progress"
+  MAVEN_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000 -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3"
+
+jobs:
```

Comment:
You could use the shared ci.yml instead: https://github.com/killbill/killbill-avatax-plugin/blob/ec2c9113fa73e442a6e70336446887c6d16f5bd1/.github/workflows/ci.yml#L9

Reply:
Great, I fixed it in https://github.com/killbill/killbill-catalog-plugin-test/pull/7/commits/e8b9422123fffba9bb4cb44be64c745b59d4f150

Reply:
Weird `verify` fails with RAT plugin:
<img width="1080" alt="Screen Shot 2022-05-27 at 6 04 30 PM" src="https://user-images.githubusercontent.com/246910/170803679-d11b2ef4-5f0b-48c3-9ab5-a913299d33a2.png">

Then I try it locally, and don't get any errors:

```
>  mvn -Ptravis verify
...
[INFO] --- apache-rat-plugin:0.13:check (default) @ catalog-test ---
[INFO] Added 1 additional default licenses.
[INFO] Enabled default license matchers.
[INFO] Added 2 custom approved licenses.
[INFO] Will parse SCM ignores for exclusions...
[INFO] Parsing exclusions from /Users/stephane/Src/killbill/killbill-catalog-plugin-test/.gitignore
[INFO] Finished adding exclusions from SCM ignore files.
[INFO] 164 implicit excludes (use -debug for more details).
[INFO] 19 explicit excludes (use -debug for more details).
[INFO] 42 resources included (use -debug for more details)
[INFO] Rat check: Summary over all files. Unapproved: 0, unknown: 0, generated: 0, approved: 41 licenses.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  22.770 s
[INFO] Finished at: 2022-05-27T18:03:37-07:00
[INFO] ------------------------------------------------------------------------

```

🤔

Reply:
It's probably because https://github.com/killbill/killbill-oss-parent/commit/d1d7bd59efd7c1bc5432329eddad21bdd195e26b isn't on master.

Trying adding `<exclude>.mvn/**</exclude>` to your `rat` exclude list.